lua/cmp_tmux/tmux.lua

```diff
@@ -50,6 +50,9 @@ end
 
 function Tmux.create_pane_data_job(self, pane, on_data, on_exit)
     local cmd = { 'tmux', 'capture-pane', '-p', '-t', pane }
+    if self.config.capture_history then
+        cmd:append('-S -')
```

Comment:
On mobile, so can't confirm this -- but is even `:append` a Lua method ? Try `table.insert(cmd, '-S -')`

Reply:
If it needs to go before `pane` in the table, just do an additional `table.insert` below this new if statement.

Reply:
`:append` is recommended by `:h lua` when extending `vim.opt` options.

But, I will try `table.insert(cmd, '-S -')` next

Reply:
I confirmed that `-S -` works before or after `-t <pane>`, so I'm ok with inserting `-S -` at the end of the `cmd` table.